src/cascadia/TerminalApp/ShortcutActionDispatch.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Is there anything else we could/should move to this file while we're at it?

Reply:
I don't really think so - did you have something in mind? At this point I think they're each pretty minimalistic classes